File: internal/models/project.go
package models

type Project struct {
	ID        string   `json:"id"`
	Name      string   `json:"name"`
	IISSites  []string `json:"iisSites"`
	APIKeys   []string `json:"apiKeys"`
	CreatedAt string   `json:"createdAt"`
	UpdatedAt string   `json:"updatedAt"`
}

File: frontend/src/types/project.ts
export interface Project {
  id: string
  name: string
  iisSites: string[]
  apiKeys: APIKey[]
  createdAt: string
  updatedAt: string
} 

export interface APIKey {
  id: string
  hash: string
  createdAt: string
  prefix: string
  suffix: string
}